src/Table.luna

```diff
@@ -267,33 +631,113 @@ class Table:
 class ChartBuilder:
     table :: Table
 
-    def plot cname1 cname2:
-        c1 = self.table.at cname1 . toColumn
-        c2 = self.table.at cname2 . toColumn
+    # Plot y versus x as markers or lines.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > plot1 = table.chart.plot "col1" "col2"
+    #
+    # `columnName1`: X column name.
+    # `columnName2`: Y column name.
+    # `return`: [] OBRAZEK
+
+    def plot columnName1 columnName2:
+        c1 = self.table.at columnName1 . toColumn
+        c2 = self.table.at columnName2 . toColumn
         Plot c1 c2 "o" Nothing
 
-    def plotDate cname1 cname2: 
-        c1 = self.table.at cname1 . toColumn
-        c2 = self.table.at cname2 . toColumn
+    def plotDate columnName1 columnName2: 
+        c1 = self.table.at columnName1 . toColumn
+        c2 = self.table.at columnName2 . toColumn
         PlotDate c1 c2
 
-    def scatter cname1 cname2: 
-        c1 = self.table.at cname1 . toColumn
-        c2 = self.table.at cname2 . toColumn
+    def scatter columnName1 columnName2: 
+        c1 = self.table.at columnName1 . toColumn
+        c2 = self.table.at columnName2 . toColumn
         Scatter c1 c2
 
-    def histogram cname:
-        c = self.table.at cname . toColumn
+    # Plot a histogram.
+    #
+    # Compute and plot histogram of x.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > plot1 = table.chart.histogram "col1"
+    #
+    # `columnName`: X column name.
+    # `return`: [] OBRAZEK
+
+    def histogram columnName:
+        c = self.table.at columnName . toColumn
         Histogram c 10 Map.empty
 
-    def kde2 cname1 cname2:
-        c1 = self.table.at cname1 . toColumn
-        c2 = self.table.at cname2 . toColumn
+    # Fit and plot a bivariate kernel density estimate.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > plot1 = table.chart.kde2 "col1" "col2"
+    #
+    # `columnName1`: First input data.
```

Comment:
Not data but column name. What happens if column specified is not numeric or does not exist?